client/catapult/extensions/zeromq/tests/ZeroMqEntityPublisherTests.cpp

```diff
@@ -126,13 +126,13 @@ namespace catapult { namespace zeromq {
 	}
 
 	TEST(TEST_CLASS, CanUseCustomListenInterface_IPv4) {
-		AssertCanUseCustomListenInterface("0.0.0.0");
```

Comment:
i would probably just conditionally disable on WIN these problematic lines, but ask @gimre-xymcity

Reply:
yea, I can keep the failing tests on windows but do you think these tests should be added?
``AssertCanUseCustomListenInterface("::", "::1")``

Reply:
please ask @gimre-xymcity

Reply:
`"::", "::1"`, won't hurt I guess